clients/benchmarks/client.cpp

```diff
@@ -289,6 +297,10 @@ struct perf_blas<T, U, std::enable_if_t<std::is_same<T, float>{} || std::is_same
                 {"symv", testing_symv<T>},
                 {"symv_batched", testing_symv_batched<T>},
                 {"symv_strided_batched", testing_symv_strided_batched<T>},
+                // L3
+                {"syrk", testing_syrk<T>},
+                {"syrk_batched", testing_syrk_batched<T>},
+                {"syrk_strided_batched", testing_syrk_strided_batched<T>},
 #if BUILD_WITH_TENSILE
```

Comment:
do you need herk, herk_batched, herk_strided_batched here?

Reply:
herk primary template parameter only supports complex so it is only in the second case.  Without the context this does look suspicious.